### ISSUE TITLE
.NET Core Console App template for RC2

### ISSUE THREAD

**Issue by rustd** [OmniSharp/generator-aspnet#690]
This is a close to final sample for RC2. Attaching the sample below
[ConsoleApp1.zip](https://github.com/OmniSharp/generator-aspnet/files/260042/ConsoleApp1.zip)

.NET CLI
 .NET Command Line Tools (1.0.0-rc2-002673) or higher
 Install Location: https://github.com/dotnet/cli#installers-and-binaries

NuGet Feed
https://www.myget.org/F/aspnetrelease/api/v3/index.json
https://api.nuget.org/v3/index.json

//cc @sayedihashimi

**Comment by peterblazejewicz** [comment]
This one does not run on OS X:
```
Expect required library libhostpolicy.dylib to be present in [/Users/piotrblazejewicz/development/ConsoleApplication/bin/Debug/netcoreapp1.0]
  - This may be because of an invalid .NET Core FX configuration in the directory.
```
Tools are updated as per instruction:
```
dotnet --info
.NET Command Line Tools (1.0.0-rc2-002485)

Product Information:
 Version:     1.0.0-rc2-002485
 Commit Sha:  d3c65aee87

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64
```

**Comment by rustd** [comment]
your tools version is old

**Comment by peterblazejewicz** [comment]
Yes, that was this. Installers do not override old version. Thanks!

**Comment by peterblazejewicz** [comment]
Closed with: 721d68d
